From: Clint Byrum <clint@ubuntu.com>
Description: makes mtr look in the standard location from the 
 mysql-cluster-testsuite-7.2 package.
Forwarded: not-needed

Index: mysql-5.5/mysql-test/lib/mtr_cases.pm
===================================================================
--- mysql-5.5.orig/mysql-test/lib/mtr_cases.pm	2011-11-14 15:35:04.238715000 -0800
+++ mysql-5.5/mysql-test/lib/mtr_cases.pm	2011-11-25 14:41:00.433887578 -0800
@@ -287,7 +287,8 @@
     else
     {
       $suitedir= my_find_dir($::basedir,
-			     ["share/mysql-test/suite",
+			     ["lib/mysql-testsuite/suite",
+			      "share/mysql-test/suite",
 			      "mysql-test/suite",
 			      "internal/mysql-test/suite",
 			      "mysql-test",
